src/utils/create-pdf.js

```diff
@@ -0,0 +1,83 @@
+import { navigate } from "@reach/router";
+
+import { PDFDocument, StandardFonts, rgb } from "pdf-lib";
+import fontkit from "@pdf-lib/fontkit";
+
+import pdfLetterDefault from "./modulo_consegna.pdf";
+
+const downloadPdf = pdfBytes => {
+  const blob = new Blob([pdfBytes], { type: "application/pdf" });
+  const blobUrl = URL.createObjectURL(blob);
+  navigate(blobUrl);
+};
+
+function customSplit(str, maxLength) {
```

Comment:
Don't mix functions definitions styles. Use arrow functions everywhere for example.